core/components/form-fields/shared/parse-fields.ts

```diff
@@ -1,6 +1,21 @@
-import { UpdateCustomerAddressInput } from '~/app/[locale]/(default)/account/(tabs)/addresses/_actions/update-address';
+import { graphql, VariablesOf } from '~/client/graphql';
+
+// Faking mutation to get input type
+// Not ideal but best way to make sure input type is up to date
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const FakeMutation = graphql(`
```

Comment:
Why don't we just import the type from `~/bigcommerce.graphql.d.ts` instead?

Reply:
👀

Reply:
Will keep as is.